### PR TITLE
doc: added 2.7 release blog

### PR DESCRIPTION
**Pull request by Serendipity96**
Changes:

doc: added 2.7 release blog

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

File: website/blog/2021-06-29-release-apache-apisix-2.7.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* gzip
+* real_ip
+* proxy_max_temp_file_size
+
+## 下载
+
+下载 Apache APISIX 2.7.0-Release
+
+* 源代码：请访问下载页面`https://apisix.apache.org/downloads/`
+* 二进制安装包：请访问安装指南 `https://apisix.apache.org/zh/docs/apisix/how-to-build/`
```
Comment (moonming) [inline website/blog/2021-06-29-release-apache-apisix-2.7.md]: please add links

Comment (moonming) [reply website/blog/2021-06-29-release-apache-apisix-2.7.md]: and we need the English version

Comment (Serendipity96) [reply website/blog/2021-06-29-release-apache-apisix-2.7.md]: ok

**Review by tokers — APPROVED** [review]

**Comment by juzhiyuan** [comment]
@Serendipity96 Please add the English version in another PR

**Pull request merged**